README.md

```diff
@@ -64,7 +64,7 @@ _Note: The version numbers below are only examples, meant to illustrate the rela
 | Versions are the same.    | 5.1.2                      | 5.1.2              | 💚 OK      |
 | ES patch number is newer. | 5.1.__2__                  | 5.1.__5__          | ⚠️ Logged warning      |
 | ES minor number is newer. | 5.__1__.2                  | 5.__5__.0          | ⚠️ Logged warning      |
-| ES major number is newer. | __5__.1.2                  | __6__.0.0          | 🚫 Fatal error      |
+| ES major number is newer. | __5__.1.2                  | __6__.0.0          | ⚠️ Logged warning      |
```

Comment:
I thought about it, but since this is a pretty special case and it means version checking is more lenient rather than less lenient, I think it's best to avoid making this table more complicated.